src/client/Client.js

```diff
@@ -414,6 +415,15 @@ class Client extends BaseClient {
       query.set('guild_id', guildID);
     }
 
+    if (options.additionalScopes) {
+      const scopes = options.additionalScopes;
+      if (scopes.some(scope => !InviteScopes.includes(scope))) {
```

Comment:
A sanity check to ensure that `options.additionalScopes` is an array could be added here which will correspond with the thrown error below that.
```suggestion
      if (!Array.isArray(scopes) || scopes.some(scope => !InviteScopes.includes(scope))) {
```

Reply:
oh wow, I can't believe I didn't already include this 🤦